Makefile

```diff
@@ -460,6 +460,14 @@ yetus:
 		--empty-patch \
 		--plugins=all \
 		--patch-dir=/src/yetus-output
+	@echo Checking Dockerfiles for inconsistencies
+	make -C tools/dockerfile-from-checker/
+	./tools/dockerfile-from-checker/dockerfile-from-checker ./ \
+		-i ./eve-tools/bpftrace-compiler/Dockerfile \
+		-i ./pkg/optee-os/Dockerfile \
+		-i pkg/bsp-imx/Dockerfile \
```

Comment:
Why optee-os and bso-imx needs to be ignored?

Reply:
tags differ for image lfedge/eve-cross-compilers:250abc77c8c39664905b66a2673102ec5cd3b056 in files /home/christoph/projects/eve/pkg/bsp-imx/Dockerfile and /home/christoph/projects/eve/pkg/pillar/Dockerfile

Reply:
tags differ for image lfedge/eve-cross-compilers:250abc77c8c39664905b66a2673102ec5cd3b056 in files /home/christoph/projects/eve/pkg/optee-os/Dockerfile and /home/christoph/projects/eve/pkg/pillar/Dockerfile

Reply:
```
pkg/bsp-imx/Dockerfile:23:FROM --platform=${BUILDPLATFORM} lfedge/eve-cross-compilers:250abc77c8c39664905b66a2673102ec5cd3b056 AS cross-compilers
pkg/optee-os/Dockerfile:20:FROM --platform=${BUILDPLATFORM} lfedge/eve-cross-compilers:250abc77c8c39664905b66a2673102ec5cd3b056 AS cross-compilers
pkg/pillar/Dockerfile:36:FROM --platform=${BUILDPLATFORM} lfedge/eve-cross-compilers:2a1d062fce410865e7024a83de327a68e52db26c AS cross-compilers
```

They shouldn't differ, they should be the same as well, like eve container.... just bump them to the newest version and you can remove from the ignore list...

Reply:
done

Reply:
@rene nope:
```
#18 [build-cross-target 2/3] RUN apk add --no-cache --allow-untrusted -X /packages build-base-"riscv64" build-base-armhf
Error: error building "lfedge/eve-optee-os:d031d1245591c210958b8cec260eaa804f05b157": error building for arch riscv64: failed to solve: process "/bin/sh -c apk add --no-cache --allow-untrusted -X /packages build-base-\"${EVE_TARGET_ARCH}\" build-base-armhf" did not complete successfully: exit code: 1
#18 0.043   build-base-armhf (no such package):
#18 0.043 ERROR: unable to select packages:
#18 0.046     required by: world[build-base-armhf]
#18 ERROR: process "/bin/sh -c apk add --no-cache --allow-untrusted -X /packages build-base-\"${EVE_TARGET_ARCH}\" build-base-armhf" did not complete successfully: exit code: 1
2024/10/11 13:02:40 error during command execution: error building "lfedge/eve-optee-os:d031d1245591c210958b8cec260eaa804f05b157": error building for arch riscv64: failed to solve: process "/bin/sh -c apk add --no-cache --allow-untrusted -X /packages build-base-\"${EVE_TARGET_ARCH}\" build-base-armhf" did not complete successfully: exit code: 1
```

Reply:
yup, you didn't bump to the latest one:

```diff
diff --git a/pkg/pillar/Dockerfile b/pkg/pillar/Dockerfile
index 17243cef9..cfc58c383 100644
--- a/pkg/pillar/Dockerfile
+++ b/pkg/pillar/Dockerfile
@@ -33,7 +33,7 @@ ARG BUILD_PKGS_BASE
 RUN BUILD_PKGS="${BUILD_PKGS_BASE}" eve-alpine-deploy.sh
 
 # hadolint ignore=DL3029
-FROM --platform=${BUILDPLATFORM} lfedge/eve-cross-compilers:2a1d062fce410865e7024a83de327a68e52db26c AS cross-compilers
+FROM --platform=${BUILDPLATFORM} lfedge/eve-cross-compilers:250abc77c8c39664905b66a2673102ec5cd3b056 AS cross-compilers
 
 # will use several packages from target arch and copy them to sysroot
 # hadolint ignore=DL3006
```

and then all eve-cross-compilers should stay with the same hash.....